mirage/serializers/node-addon.ts

```diff
@@ -0,0 +1,29 @@
+import { ModelInstance } from 'ember-cli-mirage';
+import config from 'ember-osf-web/config/environment';
+
+import NodeAddonModel from 'ember-osf-web/models/node-addon';
+
+import ApplicationSerializer from './application';
+
+const { OSF: { apiUrl } } = config;
+
+export default class NodeAddonSerializer extends ApplicationSerializer<NodeAddonModel> {
+    buildRelationships(model: ModelInstance<NodeAddonModel>) {
+        return {
+            node: {
+                links: {
+                    related: {
+                        href: `${apiUrl}/v2/nodes/${model.node.id}`,
+                        meta: this.buildRelatedLinkMeta(model, 'node'),
+                    },
+                },
+            },
```

Comment:
As discussed in Slack, adding the externalAccount relationship to the node-addon is tricky because it needs the user and the user isn't on that model. We'd have to add a user relationship to the node-addon model to do it.

Reply:
After a bit of discussion, may not be needed, as this would require that this model be aware of the `user`